community/contributing/pr_review_guidelines.rst

```diff
@@ -0,0 +1,351 @@
+  of contributions. If the pull request consists of several
+  commits, each commit must be as functional as if other
+  commits didn't exist.
+
+* **Fixes made during the review process must be squashed into
+  the main commit.**
+
+  For multi-commit PRs check that those fixes are amended in
+  the relevant commits, and are not just applied on top of
+  everything.
+
+* **Make sure that the PR has no merge conflicts.**
+
+  It must be a one click merge, and if it's not the contributor
+  needs to rebase their work from the current master.
+
+* **Check for proper commit attribution.**
+
+  This can primarily happen with new contributors, as they
+  often don't provide a correct author signature in their
+  commits (i.e. they don't use their actual email address, or
+  the address they use isn't connected to their GitHub account).
+  This can result in the PR being authored by seemingly one
+  person, but submitted for review by another. Ultimately,
+  it's up to them if they want to fix it, but such PRs won't
+  count towards their contributions to the project and will
+  keep them forever "New contributor" as far as GitHub is
+  concerned.
+
+* **Check for proper commit messages.**
+
+  While we don't have a very strict ruleset for commit messages,
+  we still require them to be short yet descriptive and use proper
+  English. As a maintainer you've probably written them enough
+  times to know how to make one, but for a general template
+  think about *"Fix <issue> in <part of codebase>"*.
+
+4. GitHub checklist
+^^^^^^^^^^^^^^^^^^^
+
+* **Validate the target branch of the PR.**
+
+  Godot development happens around the ``master`` branch. Therefore
+  most pull requests must be made against it, and can then be
+  backported to other branches. Be wary of people making PRs
+  on the version they are working on (e.g, ``3.3``) and guide
+  them to make a change against a higher-order branch. If the
+  change doesn't make sense for the ``master``, the initial PR can
+  be made against the current maintenance branch, such as ``3.x``.
+  It's okay for people to make multiple PRs for each target
+  branch, especially if the changes cannot be easily backported.
+  Cherry-picking is also an option, if possible. Use the appropriate
+  labels if the PR can be cherrypicked (e.g. ``cherrypick:3.x``).
+
+.. note::
+
+    It is possible to change the target branch of the PR,
+    that has already been submitted, but be aware of the
+    consequences. As it cannot be synchronized with the push,
+    the target branch change will inevitable tag the entire
+    list of maintainers for review. It may also render the
+    CI incapable of running properly. A push should help with
+    that, but if nothing else, recommend opening a new, fresh PR.
+
+* **Make sure that the appropriate milestone is assigned.**
+
+  This will make it more obvious which version would include the
+  submitted changes, should the pull request be merged now.
```

Comment:
```suggestion
  This will make it more obvious which version would include the
  submitted changes, and if the pull request should be merged now.
```

Reply:
This is not correct, the phrase here means "Should the pull request be merged now, it will be included in this version".